src/app/(home)/_components/CreateMeetingForm.tsx

```
@@ -0,0 +1,148 @@
+		meetingEndDate,
+		meetingPeople,
+		meetingNameTextChangeHandler,
+		meetingPlaceTextChangeHandler,
+		meetingImageTextChangeHandler,
+		meetingSelectedServiceChangeHandler,
+		meetingPeopleTextChangeHandler,
+		meetingStartDateChangeHandler,
+		meetingEndDateChangeHandler,
+	} = useMeetingForm();
+
+	console.log('meetingName : ', meetingName);
```

ESLint

[error] Unexpected console statement.

(no-console)